showkase-processor/src/main/java/com/airbnb/android/showkase/processor/models/ShowkaseMetadata.kt

```diff
@@ -153,6 +157,22 @@ internal fun ShowkaseCodegenMetadata.toModel(element: Element): ShowkaseMetadata
     }
 }
 
+private fun ShowkaseCodegenMetadata.getCodegenMetadataTypeMirror(): Pair<TypeMirror?, TypeMirror?> {
+    val enclosingClassTypeMirror = try {
+        enclosingClass
+        listOf<TypeMirror>()
+    } catch (mte: MirroredTypesException) {
+        mte.typeMirrors
+    }.firstOrNull()
+    val previewParameterClassTypeMirror = try {
+        previewParameterClass
```

Comment:
to help yourself out i'd recommend creating an extension function to get a type mirror

Reply:
oh I initially did that but it crashes as even accessing that field throws the MirroredTypeException. So I reverted that change - https://github.com/airbnb/Showkase/pull/105/commits/e7721cc65c44ce18d8a314fb9cec3c3ef318a1d9 @elihart